src/test/javascript/spec/component/programming-exercise/category-issues-chart.component.spec.ts

```diff
@@ -0,0 +1,122 @@
+import { ComponentFixture, fakeAsync, TestBed, tick } from '@angular/core/testing';
+import { CategoryIssuesChartComponent } from 'app/exercises/programming/manage/grading/charts/category-issues-chart.component';
+import { StaticCodeAnalysisCategory, StaticCodeAnalysisCategoryState } from 'app/entities/static-code-analysis-category.model';
+import { NgbTooltip } from '@ng-bootstrap/ng-bootstrap';
+import { MockDirective } from 'ng-mocks';
+
+describe('CategoryIssuesChartComponent', () => {
+    let fixture: ComponentFixture<CategoryIssuesChartComponent>;
+    let comp: CategoryIssuesChartComponent;
+    let category: StaticCodeAnalysisCategory;
+    beforeEach(async () => {
```

Comment:
```suggestion
    let category: StaticCodeAnalysisCategory;

    beforeEach(() => {
```
No need for async here.

Reply:
Thanks for the feedback, removed in aabf45a